src/Notify/Resources/doc/index.rst

```diff
@@ -0,0 +1,140 @@
+Symfony UX Notify
+===================
+
+Symfony UX Notify is a Symfony bundle providing light dropzones for
+file inputs in Symfony Forms. It is part of `the Symfony UX initiative`_.
+
+Symfony UX Notify is a Symfony bundle integrating realtime native
+notifications in Symfony applications using `Mercure`_.
+It is part of `the Symfony UX initiative`_.
+
+Installation
+------------
+
+Before you start, make sure you have `Symfony UX configured in your app`_.
+
+Then, install this bundle using Composer and Symfony Flex:
+
+.. code-block:: terminal
+
+    $ composer require symfony/ux-notify
+
+    # Don't forget to install the JavaScript dependencies as well and compile
+    $ yarn install --force
+    $ yarn encore dev
+
+Also make sure you have at least version 3.0 of
+`@symfony/stimulus-bridge`_ in your ``package.json`` file.
+
+Configuration
+-------------
+
+Symfony UX Notify can be configured to specify the Mercure hub to use:
+
+.. code-block:: yaml
+
+    # config/packages/notify.yaml
+
+    notify:
+        # Specifies the Mercure hub to use. Defaults to "mercure.hub.default"
+        mercure_hub: mercure.hub.custom
+
+
+Usage
+-----
+
+To use Symfony UX Notify you must have a `running Mercure server`_.
+
+Then, you can inject the ``NotifierInterface`` service and send messages on the ``chat/mercure`` channel::
+
+    // ...
+    use Symfony\Component\Notifier\Notification\Notification;
+    use Symfony\Component\Notifier\NotifierInterface;
+
+    class AnnounceFlashSalesCommand extends Command
```

Comment:
We should include the `#[AsCommand]` above this, just to make this a complete example.